### PR TITLE
task 9348: login screen - handle event

### PR DESCRIPTION
**Pull request by vanpd-0859**
### Issue Link :link:

https://edu-redmine.sun-asterisk.vn/issues/9348
### Goals :soccer:

Create a UI Action:
- Click Create New Wallet button to go to Create New Wallet screen
- Click Recover Exist Wallet to go to Wallet Recovery screen
- Click to how to import wallet hyperlink label to go to How to import wallet guide screen

### CODE REVIEW

**Comment by vanpd-0859** [comment]
ready

**Review by tanbtd-1747 — APPROVED** [review]

**Review by lehuudung — APPROVED** [review]

File: CryptoWallet/Scenes/Guide/GuideViewController.swift [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+//
+//  Created by Phan Dinh Van on 4/2/19.
+//  Copyright © 2019 Phan Dinh Van. All rights reserved.
+//
+
+import UIKit
+import Reusable
+
+final class GuideViewController: UIViewController, StoryboardSceneBased {
+    static var sceneStoryboard = Storyboards.main
```
Comment (longvt96) [inline CryptoWallet/Scenes/Guide/GuideViewController.swift]: tách ra 1 extension nhé 
extension GuideViewController: StoryboardSceneBased {
    static var sceneStoryboard = Storyboards.main
}

File: CryptoWallet/Scenes/Login/CreateWallet/CreateWalletViewController.swift [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+//
+//  Created by Phan Dinh Van on 4/2/19.
+//  Copyright © 2019 Phan Dinh Van. All rights reserved.
+//
+
+import UIKit
+import Reusable
+
+final class CreateWalletViewController: UIViewController, StoryboardSceneBased {
+    static var sceneStoryboard = Storyboards.main
```
Comment (longvt96) [inline CryptoWallet/Scenes/Login/CreateWallet/CreateWalletViewController.swift]: tương tự trên

File: CryptoWallet/Scenes/Login/LoginViewController.swift [hunk 3]
```diff
@@ -24,15 +24,32 @@ final class LoginViewController: UIViewController {
...
+        let createWalletController = CreateWalletViewController.instantiate()
+        navigationController?.pushViewController(createWalletController, animated: true)
     }
     
     @IBAction private func handleRecoverWalletTapped(_ sender: Any) {
+        let recoverWalletController = RecoverWalletViewController.instantiate()
+        navigationController?.pushViewController(recoverWalletController, animated: true)
+    }
+    
+    @objc private func handleImportGuideTapped(_ sender: Any) {
```
Comment (tienpx3010) [inline CryptoWallet/Scenes/Login/LoginViewController.swift]:
```suggestion
    @objc private func handleShowGuideTapped(_ sender: Any) {
```

File: CryptoWallet/Scenes/Login/RecoverWallet/RecoverWalletViewController.swift [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+//
+//  Created by Phan Dinh Van on 4/2/19.
+//  Copyright © 2019 Phan Dinh Van. All rights reserved.
+//
+
+import UIKit
+import Reusable
+
+final class RecoverWalletViewController: UIViewController, StoryboardSceneBased {
+    static var sceneStoryboard = Storyboards.main
```
Comment (longvt96) [inline CryptoWallet/Scenes/Login/RecoverWallet/RecoverWalletViewController.swift]: tương tự trên

File: CryptoWallet/Scenes/Guide/GuideViewController.swift (hunk 1 above)
Comment (tienpx3010) [inline CryptoWallet/Scenes/Guide/GuideViewController.swift]: Đưa vào extension

File: CryptoWallet/Scenes/Login/LoginViewController.swift [hunk 5]
```diff
@@ -24,15 +24,32 @@ final class LoginViewController: UIViewController {
...
+        let importGuideController = GuideViewController.instantiate()
+        navigationController?.pushViewController(importGuideController, animated: true)
     }
     
     private func configView() {
         [createWalletButton, recoverWalletButton].forEach {
             $0.setBorder(cornerRadius: 5, borderWidth: 2, borderColor: .darkGray)
         }
+        let tapImportGuideLabel = UITapGestureRecognizer(target: self, action: #selector(handleImportGuideTapped(_:)))
+        importGuideLabel.addGestureRecognizer(tapImportGuideLabel)
```
Comment (tienpx3010) [inline CryptoWallet/Scenes/Login/LoginViewController.swift]: Dùng .then

**Comment by vanpd-0859** [comment]
ready

**Review by haith-1481 — APPROVED** [review]

File: CryptoWallet/Extensions/RecoverWalletViewController+.swift [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+//  CryptoWallet
+//
+//  Created by Phan Dinh Van on 4/2/19.
+//  Copyright © 2019 Phan Dinh Van. All rights reserved.
+//
+
+import Foundation
+import Reusable
+
+extension RecoverWalletViewController: StoryboardSceneBased {
```
Comment (tienpx3010) [inline CryptoWallet/Extensions/RecoverWalletViewController+.swift]: Bỏ extension này vào cuối file RecoverWalletViewController luôn em nhé. Không cần viết riêng

**Comment by vanpd-0859** [comment]
ready

**Review by tienpx3010 — APPROVED** [review]

**Review by longvt96 — APPROVED** [review]

**Pull request merged**